Platformio/HAL/Targets/ESP32/display/display.cpp

```diff
@@ -37,16 +39,40 @@ Display::Display(int backlight_pin, int enable_pin): DisplayAbstract(),
         digitalWrite(this->mEnablePin, LOW);  // LCD Logic on
     }  
 
+    setupTFT();
     setupTouchScreen();
     mFadeTaskMutex = xSemaphoreCreateBinary();
     xSemaphoreGive(mFadeTaskMutex);
 }
 
+void Display::setupBacklight() {
+  // Configure the backlight PWM
+  // Manual setup because ledcSetup() briefly turns on the backlight
+  ledc_channel_config_t ledc_channel_left;
+  ledc_channel_left.gpio_num = (gpio_num_t)mBacklightPin;
+  ledc_channel_left.speed_mode = LEDC_HIGH_SPEED_MODE;
+  ledc_channel_left.channel = LEDC_CHANNEL_5;
+  ledc_channel_left.intr_type = LEDC_INTR_DISABLE;
+  ledc_channel_left.timer_sel = LEDC_TIMER_1;
+  ledc_channel_left.flags.output_invert = 1; // Can't do this with ledcSetup()
+  ledc_channel_left.duty = 0;
+  ledc_channel_left.hpoint = 0;
+  ledc_timer_config_t ledc_timer;
+  ledc_timer.speed_mode = LEDC_HIGH_SPEED_MODE;
+  ledc_timer.duty_resolution = LEDC_TIMER_8_BIT;
+  ledc_timer.timer_num = LEDC_TIMER_1;
+  ledc_timer.clk_cfg = LEDC_AUTO_CLK;
+  ledc_timer.freq_hz = 640;
+  ledc_channel_config(&ledc_channel_left);
+  ledc_timer_config(&ledc_timer);
+}
+
 void Display::onTouch(Notification<TS_Point>::HandlerTy aTouchHandler){
   mTouchEvent.onNotify(std::move(aTouchHandler));
 }
 
 void Display::setupTFT() {
+  delay(100);
```

Comment:
What is this delay for?

Reply:
It was always in the original code. Comment was to let the LCD Driver power on https://github.com/CoretechR/OMOTE/blob/main/Platformio/src/main.cpp#L500
Not 100% sure if it's needed. I was trying to get closer to the original code that worked well